docs/creating-elements.md

```diff
@@ -95,6 +95,16 @@ Props in the dynamic map will override props that are defined statically.
 ;; => #js {:type my-component :props #js {:a 1 :b 3}}
 ```
 
+This syntax is also useful when you want to pass a non-literal map to a component:
+
+```clojure
+(let [m {:foo "bar"}]
+  ;; This will not work; `m` will be treated as a child:
+  ($ my-component m)
+  ;; This will work; `my-component` will recieve `m` as props:
+  ($ my-component {& m}))
+```  
+
```

Comment:
I feel like this section doesn't follow from the previous, since it's essentially repeating the same information without acknowledging it.

I think that adding a warning here that passing in non-literal props _will not work_. That should be clear if people read the rest of the document, but I understand that many people will just skim this.

Reply:
Considering that defn and $ demand literal props, I think it could be explicity stated at the start of the docs. I confess I misread the native properties, piece. I've related it to react native and didnt read the Dynamic props section.